src/main/groovy/io/cify/framework/core/Device.groovy

```diff
@@ -237,7 +253,7 @@ class Device implements IDevice {
         WebDriver driver = DriverFactory.getDriver(getCapabilities())
         this.driver = driver
 
-        if (System.getProperty("videoRecord") == "true") {
+        if (getCapabilities().getCapability("videoRecord") == "true") {
```

Comment:
Why no to have all capabilities as Enum somewhere instead of using magic strings?

Reply:
Absolutely agree, but let's make it as a separate PR, as it is going to be quite a big change